build.rs

```diff
@@ -445,7 +448,14 @@ fn generate_bindings(r_paths: &InstallationPaths, version_info: &RVersionInfo) {
         match e.get_kind() {
             EnumDecl | FunctionDecl | StructDecl | TypedefDecl | VarDecl | UnionDecl => {
                 if let Some(n) = e.get_name() {
-                    allowlist.insert(n);
+                    // due to a bug in LLVM 16.0.0 on Windows, unnamed
+                    // items are included here
+                    // See
+                    // https://github.com/rust-lang/rust-bindgen/issues/2488
+                    // https://github.com/extendr/libR-sys/issues/151
```

Comment:
There were some clarifications & explanation sin the bindgen issue, can you update these comments accordingly?

Reply:
I don't know exactly what to retain. Honestly, I'm just going to reference your/our issue and just call it a day there. The use of `Entity::is_anonymous` is mentioned there.